src/model.js

```diff
@@ -424,11 +434,24 @@ const BookshelfModel = ModelBase.extend({
     let columnNames, candidates;
     if (_.isArray(arguments[1])) {
       columnNames = arguments[1];
-      candidates = _.drop(arguments, 2);
     } else {
       columnNames = null;
-      candidates = _.drop(arguments);
     }
```

Comment:
I don't know if making the registry and non-registry calls work differently is a good idea.

Making these:

```
morphTo('imageable', [Site, 'cover_photo'], [Author, 'profile_pic'])
// columnNames: null
// morphValues: [Author, 'profile_pic'] [Site, 'cover_photo']

morphTo('imageable', ['Site', 'cover_photo'], ['Author', 'profile_pic'])
// columnNames: ['Site', 'cover_photo']
// morphValues: ['Author', 'profile_pic']
```

would make migrating from a _non-registry_ code to code with **with** _registry_ more difficult.